### ISSUE TITLE
UserPropProfileMultiGet renamed to UserPropMultiGet

### ISSUE THREAD

**Issue by Denton-L** [ryanharrison554/pyovpn-as#6]
In OpenVPN 2.11.3, the UserPropProfileMultiGet method is renamed to UserPropMultiGet. We have to use the following monkeypatch in order to get the code to work:

```
pyovpn_as.api.rpc._SupportedMethod.METHODS['UserPropMultiGet'] = (
        pyovpn_as.api.rpc._SupportedMethod.METHODS['UserPropProfileMultiGet'])
pyovpn_as.api.rpc.RpcClient.UserPropProfileMultiGet = (
        lambda self, *args, **kwargs: self.UserPropMultiGet(*args, **kwargs))
```

**Comment by peritz** [comment]
I'll have to think about how I am going to deal with different API versions. I'm thinking of changing the schema of the JSON to include specific fields for methods that have changed per version, then when resolving which method to use, the RPC module will determine which to select based on the available versions in the JSON and the version given.

For example, if the developer uses version 2.11.1 then it will resolve to UserPropProfileMultiGet, but if they use any version which is >= 2.11.3 it will use UserPropMultiGet.

I'll also investigate that change, not sure why they would make a breaking change on a patch version, though the RPC might be considered not a public API so is understandable in this case.

**Comment by peritz** [comment]
This has been merged in #7
